client/src/modules/account_reference_type/templates/action.cell.html

```diff
@@ -0,0 +1,36 @@
+<div class="ui-grid-cell-contents text-right" uib-dropdown dropdown-append-to-body  data-row="{{row.entity.label}}">
+  <span ng-if="!row.entity.fixed">
+    <a href uib-dropdown-toggle data-action="open-dropdown-menu">
+      <span data-method="action" translate>FORM.BUTTONS.ACTIONS</span>
+      <span class="caret"></span>
+    </a>
+
+    <ul data-row-menu="{{row.entity.label}}" class="dropdown-menu-right" bh-dropdown-menu-auto-dropup uib-dropdown-menu>	
+      <li>
```

Comment:
Could you add a header to the dropdown to give the user context about which row they have clicked?  It should look like [the one found on the account action dropdown](https://github.com/IMA-WorldHealth/bhima/blob/master/client/src/modules/accounts/templates/grid.actionsCell.tmpl.html#L9).